### PR TITLE
9f1: Remove "nearest" from wording

### LINKED ISSUE #1106: Rule 9f1 implies truncation is similar to rounding
**Describe the typo**
The current text of the rule reads:
> 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to the **nearest** hundredth of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.

(Emphasis mine)
Truncation is closer to chopping off the least significant digits than adjusting a measurement to the nearest hundredth of a second.
Results of both `1.230` and `1.239` would be recorded as `1.23` when truncating, even though the nearest hundredth of a second to `1.239` is clearly `1.24`

**Possible solution**
Strike the word "nearest" from the text of the rule.  I have a branch with this change ready to go.

### PR DESCRIPTION
**Pull request by MFGookey**
Rule 9f1 currently says that all timed results under 10 minutes are truncated to the "nearest" hundredth of a second.  This is confusing as when you truncate, you don't apply any concepts of "nearest".  Both X.000 and X.009 would be recorded as X.00 if one were to truncate the precision at the hundredths place.  The nearest hundredth to X.009 is X.01.
This resolves #1106

### CODE REVIEW

File: wca-regulations.md
Comment (lgarron) [inline wca-regulations.md]: I'd go with this, as "the hundredth of a second" sounds a bit like it's talking specifically about `.01`, and "hundredths of a second" is a bit less overly specific.
```suggestion
    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated down to hundredths of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
```

Comment (MFGookey) [reply wca-regulations.md]: Thanks for taking a look!  I agree that just dropping the "nearest" left the phrasing a little clunky.  I appreciate your having taken a second run at the specific wording.

I like your use of the word "down" to help plant the idea that you'd not end up at 1.24 from 1.239 for example.

If you're completely happy with the phrasing here, so am I.

**Review by lgarron — APPROVED** [review]

**Comment by lgarron** [comment]
I'd like another @thewca/wrc-team member to chime in before merging, but looks good to me.

**Comment by MFGookey** [comment]
> I'd like another @thewca/wrc-team member to chime in before merging, but looks good to me.

I think that's a great idea; I was hoping that there would be a degree maintainer consensus when futzing with rules for Judges.  Thanks for being so responsive!

**Comment by Nanush7** [comment]
Looks good to me. We are still waiting for Board approval in other PRs, though.

**Comment by ohexter** [comment]
"truncated down" is superfluous and not consistent with normal usage. "truncated" on its own should be used.

**Pull request merged**
